tests/unittests/functionality/test_google_oauth.py

```diff
@@ -21,9 +21,9 @@ def test_user_already_logged_in(self):
             register(self.app, 'email@gmail.com', 'test')
             logout(self.app)
             login(self.app, 'email@gmail.com', 'test')
-            self.assertTrue('Open Event' in self.app.get('/gCallback/?state=dummy_state&code=dummy_code',
+            self.assertTrue('Open Event' in self.app.get('/gCallback/?state=dummy_state&code=dummy_code&test=1',
```

Comment:
So what does adding test=1 do? You are checking in the function if it is for test, then redirect directly? If that is what you are doing, doesn't that make the purpose of test wrong itself?